client/src/RatingsReviews/RatingBreakdown.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import axios from 'axios';
+
+class RatingBreakdown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.setInitialState = this.setInitialState.bind(this);
+  }
+
+  componentDidMount() {
+    const { productId } = this.props;
+    // Get meta data.
+    axios
```

Comment:
I think we should move this out of ComponentDidMount

Reply:
I might be wrong, but I think getting the metadata on the product within componentdidmount is within the react lifecycle spec. Based on the React docs:
componentDidMount() is invoked immediately after a component is mounted (inserted into the tree). Initialization that requires DOM nodes should go here. _If you need to load data from a remote endpoint, this is a good place to instantiate the network request._
Where would it be more applicable?